wiki/Phylo_cookbook.md

```diff
@@ -325,7 +325,7 @@ def to_adjacency_matrix(tree):
             adjmat[lookup[parent], lookup[child]] = 1
     if not tree.rooted:
         # Branches can go from "child" to "parent" in unrooted trees
-        adjmat += adjmat.transpose
+        adjmat = adjmat + adjmat.transpose()
```

Comment:
Style wise this would be preferable:

```python
adjmat += adjmat.transpose()
```

Reply:
When I had `adjmat += adjmat.transpose()`, I had originally gotten some error like

```bash
TypeError: ufunc 'add' output (typecode 'O') could not be coerced to provided
output parameter (typecode 'd') according to the casting rule ''same_kind''
```

The `typecode` values were different (I don't recall what exactly they were), but the issue was resolved by changing to `adjmat = adjmat + adjmat.transpose()` due to implicit typecasting in the addition (as opposed to the in-place addition of `+=`). I have since updated `numpy` and `biopython`, and I no longer get the error with `+=`, so I agree that it would be preferable